MLTranslate/pom.xml

```diff
@@ -3,21 +3,41 @@
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>myTranslate</groupId>
-  <artifactId>google-cloud-translation-hello-world</artifactId>
+  <groupId>io.happycoding</groupId>
```

Comment:
Looks like maybe you were referencing code from happycoding.io to get things to compile as expected? It's okay to use other people's code as a reference but you should not copy-paste if you don't have explicit permission from the original author and have attributed it appropriately. Let me know if you need any help getting it to compile correctly - we can do a remote debugging session!